src/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -85,14 +87,16 @@ internal class AuthenticationRequestParameters
         public AuthorityInfo AuthorityInfo => Authority.AuthorityInfo;
         public AuthorityEndpoints Endpoints { get; set; }
         public string TenantUpdatedCanonicalAuthority { get; set; }
-        public ICacheSessionManager CacheSessionManager{ get; set; }
+        public ICacheSessionManager CacheSessionManager { get; set; }
         public SortedSet<string> Scope { get; set; }
         public string ClientId { get; set; }
         public Uri RedirectUri { get; set; }
         public IDictionary<string, string> ExtraQueryParameters { get; }
         public string Claims { get; }
 
-        #region TODO REMOVE FROM HERE AND USE FROM SPECIFIC REQUEST PARAMETERS
+        public bool IsBrokerEnabled { get; set; }
```

Comment:
>set;  [](start = 43, length = 5)

remove setter

Reply:
I have the set for unit tests to test just the request...but can remove it. not sure how to keep the tests going at the moment.

---
In reply to: [258169812](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/881#discussion_r258169812) [](ancestors = 258169812)

Reply:
as we discussed, will make this internal for now.

---
In reply to: [258185665](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/881#discussion_r258185665) [](ancestors = 258185665,258169812)